gateway/single-cluster/global-lb-basic/README.md

```diff
@@ -0,0 +1,550 @@
+    Kind:         Gateway
+    Metadata:
+      Creation Timestamp:  2021-12-04T12:55:38Z
+      Finalizers:
+        gateway.finalizer.networking.gke.io
+      Generation:  1
+      Managed Fields:
+        API Version:  networking.x-k8s.io/v1alpha1
+        Fields Type:  FieldsV1
+        fieldsV1:
+          f:metadata:
+            f:annotations:
+              .:
+              f:kubectl.kubernetes.io/last-applied-configuration:
+          f:spec:
+            .:
+            f:addresses:
+            f:gatewayClassName:
+            f:listeners:
+        Manager:      kubectl-client-side-apply
+        Operation:    Update
+        Time:         2021-12-04T12:55:38Z
+        API Version:  networking.x-k8s.io/v1alpha1
+        Fields Type:  FieldsV1
+        fieldsV1:
+          f:metadata:
+            f:annotations:
+              f:networking.gke.io/addresses:
+              f:networking.gke.io/backend-services:
+              f:networking.gke.io/firewalls:
+              f:networking.gke.io/forwarding-rules:
+              f:networking.gke.io/health-checks:
+              f:networking.gke.io/last-reconcile-time:
+              f:networking.gke.io/ssl-certificates:
+              f:networking.gke.io/target-proxies:
+              f:networking.gke.io/url-maps:
+            f:finalizers:
+              .:
+              v:"gateway.finalizer.networking.gke.io":
+          f:status:
+            f:addresses:
+        Manager:         GoogleGKEGatewayController
+        Operation:       Update
+        Time:            2021-12-04T12:56:21Z
+      Resource Version:  149805
+      UID:               c445b6bc-e672-4f3b-b633-44f17c763bab
+    Spec:
+      Addresses:
+        Type:              NamedAddress
+        Value:             gke-gxlb-ip
+      Gateway Class Name:  gke-l7-gxlb
+      Listeners:
+        Port:      443
+        Protocol:  HTTPS
+        Routes:
+          Group:  networking.x-k8s.io
+          Kind:   HTTPRoute
+          Namespaces:
+            From:  All
+        Tls:
+          Mode:  Terminate
+          Options:
+            networking.gke.io/pre-shared-certs:  gxlb-cert
+          Route Override:
+            Certificate:  Deny
+    Status:
+      Addresses:
+        Type:   IPAddress
+        Value:  x.x.x.x
+      Conditions:
+        Last Transition Time:  1970-01-01T00:00:00Z
+        Message:               Waiting for controller
+        Reason:                NotReconciled
+        Status:                False
+        Type:                  Scheduled
+    Events:
+      Type    Reason  Age                From                   Message
+      ----    ------  ----               ----                   -------
+      Normal  ADD     86s                sc-gateway-controller  default/external-http
+      Normal  UPDATE  43s (x3 over 86s)  sc-gateway-controller  default/external-http
+      Normal  SYNC    43s                sc-gateway-controller  SYNC on default/external-http was a success
+        
+    ```
+
```

Comment:
The manifest files have not been merged intentionally because different teams/roles can create Gateway or HttpRoute. For example, the network team may use Gateway recipe to create Gateway and the individual application team can use HttpRoute recipe to create their application specific routes.

Reply:
@ravi-dalal Thanks! Totally agree with you. Would you please create a single recipe file so that folks that want to test it in a single command can also do like that? (with the link from the readme). wdyt? @boredabdel wdyt?

Reply:
@danielmarzini I have merged both the manifests in one file. Please have a look and let me know if you have any other suggestions.

Reply:
@ravi-dalal Thanks! I'll DM you during the day

Reply:
@danielmarzini I think I got it now what you were asking. I looked at your recipe directory structure and files. I can created similar structure. Please have a look.